README.md

```diff
@@ -46,6 +47,24 @@ to include the theme located in this repository at `./kitty/edelweiss.conf`.
 echo 'include ~/.config/edelweiss/kitty/edelweiss.conf' >> ~/.config/kitty/kitty.conf
 ```
 
+### terminator
+
+Add a new profile to `.config/terminator/config` or change the `background_color`, `foreground_color`
+and `palette` in your default profile.
+
+```
+[profiles]
+  [[edelweiss]]
+    background_color = "#ffffff"
+    foreground_color = "#333333"
+    show_titlebar = False
+    scrollback_lines = 10000
+    palette = "#000000:#dc2626:#16a34a:#ca8a04:#2563eb:#db2777:#0891b2:#dddddd:#767676:#f87171:#4ade80:#facc15:#60a5fa:#f472b6:#22d3ee:#ffffff"
```

Comment:
Can you post a screenshot of this in this MR? I'm not using terminator, but happy to include it. 

Also, the colors in the other themes are from the Tailwind CSS palette, see [here](https://tailwindcss.com/docs/customizing-colors).

And how they are used in the [neovim color schemes](https://github.com/timofurrer/weiss/blob/1cb489179a0c0ebae28e4796e772c08e09e3c072/nvim/lua/edelweiss/colors.lua#L3).

What do you think about using the `600` shade of each base color in this palette? And black as `#333333` - I don't like `#000000` too much, but no show stopper if you really want "black".

Reply:
i tried to copy all the colors from kitty so it's just color0-color15

```
#: black (non-tailwind)
color0 #000000
color8 #767676
```

so i would just keep it consistent up to you if you want to change 
i personally didn't see a problem yet with full black


and it looks something like this for me

![image](https://github.com/timofurrer/edelweiss/assets/1682954/64f9c2ea-2265-4ca5-b72e-9948c8fbe871)

Reply:
Okay, so these are the proper colors then I suppose ...